File: android/rctmapboxar/src/main/java/com/mapbox/react/ar/math/Face3.java
package com.mapbox.react.ar.math;

public class Face3 {
    public int a;
    public int b;
    public int c;

    public Vector3 normal;
    public Vector3[] vertexNormals;

    public Face3(int a, int b, int c, Vector3[] vertexNormals) {
        this.a = a;
        this.b = b;
        this.c = c;
        this.normal = new Vector3();
        this.vertexNormals = vertexNormals;
    }
}

File: ios/RNMapboxAR/Face3.m
//
//  Face3.m
//  RNLocateTest
//
//  Created by Dave Prukop on 4/26/18.
//  Copyright © 2018 Facebook. All rights reserved.
//

#import "Face3.h"

@implementation Face3

- (instancetype)initWithA:(int)a andB:(int)b andC:(int)c andVertexNormals:(NSArray<Vector3 *> *)vertexNormals
{
  self = [super init];
  if (self) {
    self.a = a;
    self.b = b;
    self.c = c;
    self.normal = [[Vector3 alloc] init];
    self.vertexNormals = vertexNormals;
  }
  return self;
}

@end
